docker_signal.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
Ah, you have to code for it to support the clean shutdown. That might be the better way. My preference is to always require as few wrapper scripts as possible. What's your thought on the matter?